scipy/io/tests/test_wavfile.py

```diff
@@ -361,6 +361,15 @@ def test_read_incomplete_chunk():
                 wavfile.read(fp, mmap=mmap)
 
 
+def test_read_inconsistent_header():
+    # File header's size fields contradict each other
+    for mmap in [False, True]:
+        filename = 'test-8000Hz-le-3ch-5S-24bit-inconsistent.wav'
```

Comment:
Couldn't we have a binary string instead of having the need to add a file?

Reply:
I'm just following the example of all the other test files.  Probably good to test it the way users are going to, though?

Reply:
You're the expert here if it makes sense sure. I will merge once you confirmed.

Reply:
Well it's the same as all the other examples, so if we're going to change it to strings, that should be done in another PR.

Reply:
Ok, fine with me. I am merging then :) Thanks.